README.md

```diff
@@ -1 +1,41 @@
-# Emby.Theater.Electron
+# Jellyfin Theater
+
+Jellyfin Theater is a local client that connects to a Jellyfin server. 
+Currently this takes place by contacting https://
+# Building
+
+This application is implemented as an electron app and is built off of 
+a NodeJS code base. Because of this you will need to NodeJS runtime and
+package manager. The following versions have been tested: 
+
+| Software Name | Version          |
+| ------------- | ---------------- |
+| Node JS       | 11.5.0           |
+| npm           | 6.4.1            |
+| ------------- | ---------------- |
+
+
+Take the following steps to build the application
+
+```
+# Get the source
+git clone git@github.com:jellyfin/jellyfin-theater-electron.git
+cd jellyfin-theater-electron
+
+# Install dependancies for electron
```

Comment:
I think it's worth noting that the instructions are Debian- (or Ubuntu-) specific.